google/cloud/aiplatform/prediction/local_endpoint.py

```diff
@@ -376,7 +376,7 @@ def predict(
         """
         if self.container_is_running is False:
             raise RuntimeError(
-                "The local endpoint is not serving traffic. Please call `serve()`."
+                "The local endpoint is not serving traffic. Please call ``serve()``."
```

Comment:
You may need to update the unit tests if you update the code.

Reply:
Good catch! Didn't mean to modify this.